### ISSUE TITLE
Failed to execute 'setAttributeNS' on 'Element': '' - on xmlns:v="..."

### ISSUE THREAD

**Issue by sod** [angular/angular#14865]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report
```

**Current behavior**
given this template (a breadcrumb with rdf attributes for search engines):
```twig
<ol class="breadcrumb" xmlns:v="http://rdf.data-vocabulary.org/#">
    <li typeof="v:Breadcrumb">
        <a href="/foo" rel="v:url" property="v:title">foo</a>
    </li>
</ol>
```

angular 2.4.9 - works: https://plnkr.co/edit/1Wyd4n1t8RPw7JTC4nYe?p=preview
angular 4.0.0-rc.2 - throws: https://plnkr.co/edit/qBc4ODqbBkPlMDrQVNf8?p=preview

Error:
![image](https://cloud.githubusercontent.com/assets/905328/23505392/cc62cbb6-ff43-11e6-8b41-b182956d4606.png)

**Comment by sod** [comment]
The state right before the error happens:

![image](https://cloud.githubusercontent.com/assets/905328/23516933/0a01fa70-ff6f-11e6-88cb-b51d6767fb79.png)

Maybe just add `xmlns: "http://www.w3.org/1999/xhtml"` to [NAMESPACE_URIS](https://github.com/angular/angular/blob/126fda2613b1ad87ff62591bb1d3a23cd8f369a1/modules/%40angular/platform-browser/src/dom/dom_renderer.ts#L17)?

https://en.wikipedia.org/wiki/XML_namespace#Namespace_declaration

**Comment by istiti** [comment]
wonder why this isn't fixed in rc3 ? 

cc @DzmitryShylovich

**Comment by DzmitryShylovich** [comment]
@istiti idk. I don't merge anything :)
ping @vicb

**Comment by markgoho** [comment]
Still getting this message with inline SVG in rc4

```
ERROR DOMException: Failed to execute 'setAttributeNS' on 'Element': '' is an invalid namespace for attributes.
    at EmulatedEncapsulationDomRenderer2.DefaultDomRenderer2.setAttribute (http://localhost:4200/vendor.bundle.js:31240:16)
    at DebugRenderer2.setAttribute (http://localhost:4200/vendor.bundle.js:12828:23)
    at createElement (http://localhost:4200/vendor.bundle.js:9106:22)
    at createViewNodes (http://localhost:4200/vendor.bundle.js:11602:44)
    at callViewAction (http://localhost:4200/vendor.bundle.js:12014:13)
    at execComponentViewsAction (http://localhost:4200/vendor.bundle.js:11953:13)
    at createViewNodes (http://localhost:4200/vendor.bundle.js:11671:5)
    at callViewAction (http://localhost:4200/vendor.bundle.js:12014:13)
    at execComponentViewsAction (http://localhost:4200/vendor.bundle.js:11953:13)
    at createViewNodes (http://localhost:4200/vendor.bundle.js:11671:5)
    at createRootView (http://localhost:4200/vendor.bundle.js:11549:5)
    at callWithDebugContext (http://localhost:4200/vendor.bundle.js:12657:42)
    at Object.debugCreateRootView [as createRootView] (http://localhost:4200/vendor.bundle.js:12134:12)
    at ComponentFactory_.create (http://localhost:4200/vendor.bundle.js:9521:46)
    at ComponentFactoryBoundToModule.create (http://localhost:4200/vendor.bundle.js:3420:29)
```

**Comment by DzmitryShylovich** [comment]
@markgoho will be fixed in the next release

**Comment by markgoho** [comment]
Fantastic!

**Comment by istiti** [comment]
Awesome !

Cc @DzmitryShylovich

**Comment by istiti** [comment]
what mean next release, still not fixed  in rc5, :@

**Comment by trotyl** [comment]
@istiti rc.5 is not a real release, just the hot-fix of rc.4 because of wrong dependency version, you can check it at ChangeLog.

**Comment by istiti** [comment]
@trotyl  rc6 is not real release too ? 

please can this be fixed, actually can't build, regression !

cc @IgorMinar  @vicb

**Comment by markgoho** [comment]
Confirmed, still not working in rc6

**Comment by vicb** [comment]
Should be fixed by https://github.com/angular/angular/pull/14874 - probably next release

**Comment by markgoho** [comment]
Fixed in 4.0.0 final!
